modules/local/coverage_plot.nf

```diff
@@ -0,0 +1,27 @@
+process COVERAGE_PLOT{
+    tag "$sample_name - Segment:$segment - Ref Accession ID:$id"
+    label 'process_medium'
```

Comment:
I think the default process allocation (1 core, 1GB) should be enough for this process. Either that or `process_low` since this is a single-threaded low memory process.